rnd/docker-compose.yml

```diff
@@ -161,41 +167,41 @@ services:
       migrate:
         condition: service_completed_successfully
     environment:
-      - DATABASE_URL=postgresql://agpt_user:pass123@postgres:5432/agpt_local?connect_timeout=60&schema=market
-    ports:
-      - "8015:8015"
-    networks:
-      - app-network
-      
-  frontend:
-    build:
-      context: ../
-      dockerfile: rnd/autogpt_builder/Dockerfile
-      target: dev
-    develop:
-      watch:
-        - path: ./
-          target: rnd/autogpt_builder/
-          action: rebuild
-    depends_on:
-      postgres:
-        condition: service_healthy
-      rest_server:
-        condition: service_started
-      websocket_server:
-        condition: service_started
-      migrate:
-        condition: service_completed_successfully
-    environment:
+      - SUPABASE_URL=http://kong:8000
+      - SUPABASE_JWT_SECRET=your-super-secret-jwt-token-with-at-least-32-characters-long
+      - SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJhbm9uIiwKICAgICJpc3MiOiAic3VwYWJhc2UtZGVtbyIsCiAgICAiaWF0IjogMTY0MTc2OTIwMCwKICAgICJleHAiOiAxNzk5NTM1NjAwCn0.dc_X5iR_VP_qT0zsiyj_I_OZ2T9FtRU2BBNWN8Bu4GE
       - DATABASE_URL=postgresql://agpt_user:pass123@postgres:5432/agpt_local?connect_timeout=60
-      - NEXT_PUBLIC_AGPT_SERVER_URL=http://localhost:8000/api
-      - NEXT_PUBLIC_AGPT_WS_SERVER_URL=ws://localhost:8001/ws
-      - NEXT_PUBLIC_AGPT_MARKETPLACE_URL=http://localhost:8015/api/v1/market
     ports:
-      - "3000:3000"
+      - "8015:8015"
     networks:
       - app-network
 
+#  frontend:
+#    build:
```

Comment:
Leaving it commented out as I do want to figure out later on why this doesn't work

Reply:
so prod doesn't even use this docker-compose ?